saleor/graphql/checkout/mutations.py

```diff
@@ -77,6 +77,16 @@ def check_lines_quantity(variants, quantities):
     return errors
 
 
+def get_validated_address_data(checkout, errors, user, address_data):
```

Comment:
It's just a feeling, but shouldn't it be part of the checkout mutation class? Or maybe even `18nMixin`? This function is not being used elsewhere but during the checkout process, and maybe in general internalization process and `checkout` object is being called as a `cls` below, which for me is a little counterintuitive.